run_node.sh

```diff
@@ -66,7 +79,14 @@ exec /usr/bin/wakunode\
   --store=true\
   --store-message-db-url="postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@postgres:5432/postgres"\
   --store-message-retention-policy=time:86400\
+  --rln-relay=true\
+  --rln-relay-dynamic=true\
+  --rln-relay-eth-contract-address="${RLN_RELAY_CONTRACT_ADDRESS}"\
+  --rln-relay-eth-client-address="${ETH_CLIENT_ADDRESS}"\
+  --rln-relay-tree-path="/etc/rln_tree"\
   ${DNS_WSS_CMD}\
   ${NODEKEY}\
+  ${RLN_RELAY_CRED_PATH}\
+  ${RLN_RELAY_CRED_PASSWORD}\
```

Comment:
Little nitpick. Should we have all the "rln" items together?

```suggestion
  ${RLN_RELAY_CRED_PATH}\
  ${RLN_RELAY_CRED_PASSWORD}\
  ${DNS_WSS_CMD}\
  ${NODEKEY}\
```

Reply:
> Little nitpick. Should we have all the "rln" items together?

Oh yes! I saw the optional parameters as if there were in a different category, but with what we currently have we can change the order and have the RLN mandatory parameters be next to the RLN optional ones. Will fix it now